charter.md

```diff
@@ -85,15 +93,123 @@ are replacing, regardless of the length of that remainder.
 
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 4 = 1.75
+* 1.75 * 3 = 5.25
+* Round up to the nearest whole number (6)
+* Six (6) affirmative votes would be required to remove a member through a vote
```

Comment:
Convert the last bullet into a sentence starting with "So this translates to Six"

Reply:
Leaving as-is, as this section matches the other vote `Example`s elsewhere in the charter.